### PR TITLE
Specific disbursal for singular mode

### PR DESCRIPTION
**Pull request by t0xic0der**
**Description**
This PR fixes https://github.com/t0xic0der/syngrafias/issues/108.

**Have you read the [Contributing Guidelines](https://github.com/t0xic0der/syngrafias/blob/master/CONTRIBUTING.md) before opening this PR**
Yes.

**Notes for Reviewers**
This addition somehow breaks the event logging display for the Unity Mode. Don't get me wrong - events do get logged actively but they can be seen only once when you click on the "View activities" option. On subsequent invoking of that option, no activities would show up even when logging is active. You can check the logging using developer tools.

I am not quite able to figure out as to why this is happening. The event logging in the cellular mode has an identical syntax and usage, and works just fine but the one at the unity mode does not. I could use some assistance in tracking down the fault behind this. Tagging for reference. @VaibhavSaini19 @s-katte @shivangswain.

### CODE REVIEW

**Comment by t0xic0der** [comment]
Please review carefully and diagnose the listed issue. Thanks.

File: templates/singular.html [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+    <div class="ui mini modal" id="actilogs">
+        <div class="header textbase cardinal" style="padding: 2.5%; background-color: #294172; color: #ffffff;">
+            Track activities
+        </div>
+        <div id="actijuxt" class="scrolling content" style="padding: 2.5%;">
+            <p class="textbase" style="line-height: 1.25; text-align: justify; font-size: 15px;">
+                All activities including cell creation, editing, renaming and removal would show up here - unless
+                you wipe the history clean.
+            </p>
+        </div>
```
Comment (VaibhavSaini19) [inline templates/singular.html:423]: The reason behind the mentioned bug is that you have 2 divs with id `actijuxt`. The first time the user tries to track the activity, the data is logged in the correct modal, but in subsequent tracking, it gets logged to the other div (which I believe is for the `Help and support` modal). Changing the id of the other div fixes the bug.

**Review by VaibhavSaini19 — CHANGES_REQUESTED** [review]

File: templates/singular.html (hunk 1 above)
Comment (t0xic0der) [reply templates/singular.html:423]: Ohhh got it! Thanks @VaibhavSaini19 - you're a lifesaver. :fire:

**Comment by t0xic0der** [comment]
The most recent commit https://github.com/t0xic0der/syngrafias/pull/109/commits/7d39f9881a458f79a73e2e56315e1db387105196 should fix the display and handle malformed JSON documents' exception.

**Review by VaibhavSaini19 — APPROVED** [review]
LGTM

**Comment by t0xic0der** [comment]
Thanks for the review. :smile:

**Review by shivangswain — COMMENTED** [review]
Now works good on my end too.

**Pull request merged**